chatty-clownfish/bangazon-sprint-2
# rh-navupdateandmore

## Link to Ticket

in progress: #14  1/2
https://github.com/chatty-clownfish/bangazon-sprint-2/issues/34

Closes # [Link]()

## Description of Proposed Changes

-Bootstrapped navbar including cart image which lives in new static image folder
-Added links to product detail in products by type list (ticket #14).
- added login required decorator to 'my products'
-made new product_type_view.py for product type function, and removed this function from views.py. Made necessary alteration to init file


## Steps to Test

-navbar should be bootstrapped
-sell, my products and my profile buttons should appear before login, when pressed should redirect to login page and then go to requested page after logging in
- products_by_type method should now be in its own view product_type_view
-Check that producttype page products link to correct product detail
-make sure dillon's payment changes still work

Outline the steps to test

```sh
git fetch --all
git checkout rh-navupdateandmore
```

## Impacted Areas in Application

List general components of the application that this PR will affect:

views.py
product_type_view
navbar.html
products_by_type.html

* 

## Mentions @username

Tag users that need to review this code
